packages/dx-react-grid/docs/guides/column-reordering.md

```diff
@@ -15,21 +15,18 @@ Note that the [plugin order](./plugin-overview.md#plugin-order) is important.
 
 ## Column Reordering Setup
 
-Import the plugins listed above to set up a simple Grid with column reordering enabled.
+Import the plugins listed above to set up a simple Grid with column reordering enabled. Note that the [TableColumnReordering](../reference/table-column-reordering.md) plugin should be placed after the [TableView](../reference/table-view.md) plugin.
```

Comment:
Not sure that this note should be there. Seems like it should be listed in breaking change description.

Reply:
I'm going to edit the list of breaking changes anyway. But this is the guide, so the note may be useful for the first coming users. That is why I added it.

Reply:
Typically, we add a note about plugin order importance. And assume that this info is in reference. Let's write consistent guides.